game/game/playercontrol.cpp

```diff
@@ -820,7 +846,7 @@ void PlayerControl::implMove(uint64_t dt) {
     pl.setAnim(ani);
     }
 
-  setAnimRotate(pl, rot, ani==Npc::Anim::Idle ? rotation : 0, ctrl[KeyCodec::RotateL] || ctrl[KeyCodec::RotateR], dt);
+  setAnimRotate(pl, rot, ani==Npc::Anim::Idle ? rotation : 0, movement.turnRight.any(), dt);
```

Comment:
Naming `turnRight` is somewhat confusing, makes reader thing, that there is `turnLeft` somewhere. Maybe simply `turn`?